include/handlers.hpp

```diff
@@ -65,9 +66,21 @@ class BillingHandler:
     }
     else
     {
-      return &HttpStack::DEFAULT_SAS_LOGGER;
+      if (_http_acr_logging)
+      {
+        // Include bodies in ACR HTTP messages logged to SAS.
+        return &HttpStack::DEFAULT_SAS_LOGGER;
+      }
+      else
+      {
+        // Ommit bodies from ACR HTTP messages logged to SAS.
+        return &HttpStack::PRIVATE_SAS_LOGGER;
```

Comment:
So I've never seen private SAS logger before but... I assume we have it because not logging the body somehow keeps some information private. But here you're not using it for privacy reasons, you're using it because it happens to do the right thing. Worth a comment at least (but ideally we'd rename it to NO_BODY_SAS_LOGGER or something).